tests/SprykerEcoTest/Zed/Unzer/Business/CheckIsAuthorizeCanceledOmsConditionTest.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEcoTest\Zed\Unzer\Business;
```

Comment:
I still don't see folder I requested to add

Reply:
Fixed.